packages/esm-patient-orders-app/src/components/order-price-details.test.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import { screen } from '@testing-library/react';
+import OrderPriceDetailsComponent from './order-price-details.component';
+import { useOrderPrice } from '../hooks/useOrderPrice';
+import { renderWithSwr } from 'tools';
+import { useTranslation } from 'react-i18next';
+import { mockOrderPriceData } from '../../../../__mocks__/order-price-data.mock';
+
+jest.mock('../hooks/useOrderPrice');
+jest.mock('react-i18next', () => ({
+  useTranslation: jest.fn(),
+}));
+
+const mockUseTranslation = useTranslation as jest.Mock;
```

Comment:
We shouldn't need to mock react-i18next in a test. We already do that [here](https://github.com/openmrs/openmrs-esm-patient-chart/blob/main/jest.config.js#L19) with a `moduleNameMapper` entry in the root Jest config.